server/server/app.py

```diff
@@ -14,7 +16,7 @@
     <body>
```

Comment:
Why do we need HTML ?

Reply:
we do not, i was using it for testing. forgot to remove it